src/web-ui/src/public/Main.vue

```diff
@@ -43,11 +48,14 @@ export default {
   components: {
     Layout,
     RecommendedProductsSection,
+    DemoGuideBadge,
   },
   data() {
     return {
       feature: EXPERIMENT_FEATURE,
       isLoadingRecommendations: true,
+      featuredProductsDemoGuideBadgeArticle: Articles.PERSONALIZED_RANKING,
```

Comment:
I don't think we're using personalized ranking with featured products here (yet). We're just displaying the featured products that are returned from the Products service. Since we lost the "Featured" pseudo category in the category list, we need to get ranking applied here (or somewhere). That was a powerful demo feature that is no longer in the app. Is there a reason why we can't have "Featured" back in the category list/dropdown too?

Reply:
I don't see a problem with putting the Featured pseudo category in the dropdown. The dropdown button is actually going to be called "Explore" in an upcoming PR in order to more clearly show that has items other than categories (such as IVS). I can do this as part of that upcoming PR early this week.